src/attr_file.c

```diff
@@ -633,6 +633,11 @@ int git_attr_fnmatch__parse(
 		if (--spec->length == 0)
 			return GIT_ENOTFOUND;
 
+	/* Remove trailing spaces. */
```

Comment:
Doesn't this make [5cb6a2c#L216-225](https://github.com/libgit2/libgit2/commit/5cb6a2c946a071d43be3cdabf2f6d4417a4be694#diff-2aa744cf46a2b9ef04c23ad8fc42b3ceR216) redundant?